frontend/src/v5/ui/routes/viewer/toolbar/selectionToolbar/selectionToolbar.component.tsx

```diff
@@ -20,16 +20,14 @@ import EyeShowIcon from '@assets/icons/viewer/eye_show.svg';
 import EyeIsolateIcon from '@assets/icons/viewer/eye_isolate.svg';
 import { formatMessage } from '@/v5/services/intl';
 import { GroupsActionsDispatchers, TreeActionsDispatchers } from '@/v5/services/actionsDispatchers';
-import { GroupsHooksSelectors, TicketsCardHooksSelectors, TreeHooksSelectors } from '@/v5/services/selectorsHooks';
+import { TreeHooksSelectors, ViewerGuiHooksSelectors } from '@/v5/services/selectorsHooks';
 import { isEmpty } from 'lodash';
 import { FormattedMessage } from 'react-intl';
-import { Section, Container, ClearButton, ClearIcon } from './sectionToolbar.styles';
+import { Section, Container, ClearButton, ClearIcon } from './selectionToolbar.styles';
 import { ToolbarButton } from '../buttons/toolbarButton.component';
 
 export const SectionToolbar = () => {
```

Comment:
Component name should also be changed to `SelectionToolbar`

Reply:
`Section` refers literally to a "section" of the component. Each section is then separated by a vertical bar, so the name is actually correct

Reply:
But the filename is SelectionToolbar and in this PR you've corrected the name of the styles file